content/index.de.md

```diff
@@ -0,0 +1,248 @@
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Weitere Infos zum barrierefreien Gestalten Ihrer Webseite" class="simple aside" %}
+{:/}
+
+-   Eine Einführung in die Anforderungen an Barrierefreiheit und internationale Standards finden Sie unter [[Grundlagen der Barrierefreiheit]](/fundamentals/accessibility-principles/).
+-   Um einige übliche Barrieren aus Sicht des Testens zu verstehen, beachten Sie [[Einfache Checks – Eine erste Beurteilung]](/test-evaluate/preliminary/).
+-   Für ein paar grundsätzliche Überlegungen im Bezug auf das Gestalten, Schreiben und Entwickeln für Barrierefreiheit, beachten Sie [[Tipps für den Einstieg]](/tips/).
+-   Sobald Sie bereit sind, mehr über das Entwickeln und Gestalten zu erfahren, werden Sie wahrscheinlich Ressourcen wie die nachfolgenden nutzen:
+    -   [Wie erfülle ich die WCAG (Kurzübersicht)](https://www.w3.org/WAI/WCAG22/quickref/)
+    -   [Tutorials für barrierefreies Web](/tutorials/)
+-   Für Projektmanagement und organisationelle Überlegungen, lesen Sie [[Planung und Management für barrierefreies Web]](/planning-and-managing/).<br>
+    Falls Sie sofort schnelle Lösungen benötigen, finden Sie hier [[Ansätze für vorübergehende Korrekturen]](/planning/interim-repairs/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+	
+## Barrierefreiheit beurteilen {#evaluate}
+
+Beim Entwickeln und Neugestalten einer Webseite sollten Sie schon früh und während des gesamten Entwicklungsprozesses die Barrierefreiheit beurteilen, um Probleme dann zu erkennen, wenn sie einfacher zu adressieren sind. Einfache Maßnahmen wie das Ändern von Browsereinstellungen können Ihnen helfen, einige Aspekte von Barrierefreiheit direkt einzuschätzen. Eine umfangreiche Evaluierung, um zu überprüfen, ob eine Webseite alle Richtlinien für Barrierefreiheit erfüllt, erfordert mehr Aufwand.
+
+Es gibt Tools, die bei der Auswertung unterstützen. Es kann jedoch kein Tool alleine feststellen, ob eine Seite die Richtlinien für Barrierefreiheit erfüllt. Eine Auswertung durch jemand mit der entsprechenden Erfahrung ist notwendig, um zu bestimmen, ob eine Seite tatsächlich barrierefrei ist.
+
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Weitere Infos zur Beurteilung von Barrierefreiheit" class="simple aside" %}
+{:/}
+
+-   Ressourcen zur Unterstützung bei der Beurteilung von Barrierefreiheit werden in [[Barrierefreiheit von Webseiten beurteilen]](/test-evaluate/) beschrieben.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{% include excol.html type="start" id="examples" %}
+
+## Beispiele
+
+{% include excol.html type="middle" %}
+
+### Alternativer Text für Bilder
+
+![Bild von einem Logo; HTML Markup img alt='Web Accessibility Initiative logo'](https://www.w3.org/WAI/intro/alt-logo.png){:.right}
+
+Bilder sollten *[äquivalenten alternativen Text](https://www.w3.org/WAI/WCAG22/Understanding/text-alternatives)* (sogenannten alt-Text) im Code enthalten.
```

Comment:
Is there a translation of "markup" in German?

The original text reads "Images should include equivalent alternative text (alt text) in the **markup**/code."

Reply:
Not really, or at least none that is commonly used, as far as I know. I would translate it to "Markup/Code" then.